### ISSUE TITLE
unzip: short read

### ISSUE THREAD

**Issue by edersong** [captn3m0/outliner#8]
Hello,
I'm trying to run this container to perform periodically exports to a local folder, but it's failing to start reporting the following error:

```shell
unzip: short read
``` 
My command is the following (masking some private information):

```shell
docker run -e OUTLINE_BASE_URI=https://outline.example.com -e OUTLINE_TOKEN=ol_api_1234567890poiuytrewq -v /homes/markdown:/data captn3m0/outliner export /data
```

**Comment by dhx** [comment]
This should be fixed with #9

**Comment by captn3m0** [comment]
Resolved with #9 merge, and released with https://github.com/captn3m0/outliner/releases/tag/v1.0.2
